libs/frontend/feature-render-builder/src/lib/dnd/builder-dnd-component.ts

```diff
@@ -36,43 +31,50 @@ const renderImageHover = (
   site: ISite,
   component: IComponent,
   props: Record<string, unknown>,
-  componentClass: string[],
   content: IContent,
 ): VNode => {
   const imgElement = document.getElementById(component.id)
+  const buildContentWindowInner = document.getElementById(buildContentWindowInnerId)
 
-  let [width, height] = ['', '']
+  let position = '',
+    top = '',
+    right = '',
+    width = '',
+    height = '',
+    margin = ''
 
-  const { margin } = findStyles(
-    [Css.Margin],
-    site,
-    component,
-    descSortedBreakpoints.value,
-    activeBreakpoint.value,
-  )
+  if (imgElement) {
+    const imgComputedStyle = getComputedStyle(imgElement)
+
+    position = imgComputedStyle.position
+    top = imgComputedStyle.top
+    right = imgComputedStyle.right
 
-  const { width: rectWidth = 0, height: rectHeight = 0 } =
-    imgElement?.getBoundingClientRect() ?? {}
+    if (position === 'absolute' && buildContentWindowInner) {
```

Comment:
Hi @sampullman 

The good news is the absolute position `<img>` seems okay now. However, the bad news is `<img>` will now shifts a little when selected (whether it's absolute position or not 😕). Hover on another component seems to put it back to the correct position, though.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/700cf6bb-c939-4653-aabb-917b31672b3b

Some more serious cases:

https://github.com/samatechtw/pubstudio-builder/assets/59676941/4c1e5c34-50c0-4b78-9a1a-cdb1761d7175

From the video we can also see the new selection overlay is not always in the correct position. I'll leave a comment to describe related issues below.

Reply:
I see! The first case isn't so bad, but the second is a little too obvious.